README.md

```diff
@@ -171,3 +434,19 @@ $ cat > templates/xml/my_mentoring_scenario.xml
 
 Restart the workbench to take the new scenarios into account.
 
+Upgrading from Version 1
+------------------------
+
+To upgrade a course from the earlier version of this XBlock, run the following
+command on a system with edx-platform and xblock-mentoring installed:
+
+```bash
+$ SERVICE_VARIANT=cms DJANGO_SETTINGS_MODULE="cms.envs.devstack" python -m mentoring.v1.upgrade "Org/Course/Run"
```

Comment:
You need to add the "v1" directory to setup.py for this to work when the package is installed instead of a direct link to the source repo.

Also, when I tried this on a freshly imported course (cf http://sandbox.opencraft.com:18010/course/course-v1:HarvardX+GSE1.1x+3T2014 ), it gave:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)   
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/edx/app/edxapp/venvs/edxapp/lib/python2.7/site-packages/mentoring/v1/upgrade.py", line 75, in <module>
    print(" ➔ Found course: {}".format(course.display_name))
AttributeError: 'NoneType' object has no attribute 'display_name'
```

Btw, does that work on the old modulestore? The existing courses all use it.